runtime/src/append_vec.rs

```diff
@@ -175,7 +175,7 @@ impl AppendVec {
     }
 
     #[allow(clippy::mutex_atomic)]
-    fn new_empty_map(current_len: usize) -> Self {
+    pub(crate) fn new_empty_map(current_len: usize) -> Self {
         let map = MmapMut::map_anon(1).expect("failed to map the data file");
 
         AppendVec {
```

Comment:
Maybe [`impl Serialize for AppendVec`](https://github.com/solana-labs/solana/pull/9980/files#diff-657d1e7648a9fb74cb1582a9c48beb4bL484) isn't needed anymore?

Reply:
Thanks, there were vestigial serde trait implementations on AccountStorageEntry and AppendVec that will be gone entirely in the next commit.